proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -190,6 +196,13 @@ message Method {
   string description = 2;
   MethodRequestResponse request = 3;
   MethodRequestResponse response = 4;
+  MethodOptions method_options = 5;
+}
+
+// MethodOptions provides information for the documentation of options for a method.
+message MethodOptions {
+  bool deprecated = 1;
+  int32 idempotency_level = 2;
```

Comment:
This should have a comment saying that it holds a value of the enumeration `google.protobuf.MethodOptions.IdempotencyLevel`.

Reply:
Good idea. Added. Thanks!